README.md

```diff
@@ -60,10 +60,18 @@ pod 'OpenSSL-Universal'
 
 ### Carthage
 
+* If building from source is preferred:
+
 ```
 github "krzyzanowskim/OpenSSL"
 ```
 
+* If using a prebuilt framework is preferred:
+
+```
+binary "https://raw.githubusercontent.com/krzyzanowskim/OpenSSL/master/OpenSSL.json"
```

Comment:
is that right?

Reply:
If this PR gets merged, I think so. Do you see something wrong?

Reply:
I'm not familiar with this file. Should it contain all versions for Carthage?

Reply:
Yes. The json is explained here: https://github.com/Carthage/Carthage/blob/master/Documentation/Artifacts.md#binary-project-specification

So for OpenSSL, if the releases attach a zip with the xcframework, then the content could be:
```
{
    "1.1.180": "https://github.com/krzyzanowskim/OpenSSL/releases/download/1.1.180/OpenSSL.xcframework.zip",
    "1.1.1100": "https://github.com/krzyzanowskim/OpenSSL/releases/download/1.1.1100/OpenSSL.xcframework.zip"
}
```

One example of other project using this in this way is https://github.com/onfido/onfido-ios-sdk

Reply:
I see. In the past (AFAIR) Carthage was able to figure binaries from github releases automatically. Is it no longer available for xcframeworks?

Reply:
Consumers can specify a version like this in their Cartfiles if needed:

```
binary "https://raw.githubusercontent.com/krzyzanowskim/OpenSSL/master/OpenSSL.json" == 1.1.180
```

Reply:
> I see. In the past (AFAIR) Carthage was able to figure binaries from github releases automatically. Is it no longer available for xcframeworks?
Yes, it is still able to do it. But such behavior has a fallback to building from source, and users can use `--no-use-binaries` to force building from source.

This offers an alternative way which allows consumers to use `--no-use-binaries`, while still consuming `OpenSSL` as binary. This can be helpful because Carthage does not allow to specify building from source or not per dependency, see: https://github.com/Carthage/Carthage/issues/1706